src/ResourceManager/Relay/Commands.Relay/Cmdlets/AuthorizationRule/SetAzureRelayAuthorizationRules.cs

```diff
@@ -0,0 +1,128 @@
+            HelpMessage = "Namespace Name.")]
+        [Parameter(ParameterSetName = WcfRelayAuthoRuleParameterSet)]
+        [Parameter(ParameterSetName = HybridConnectionAuthoRuleParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string Namespace { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2, ParameterSetName = WcfRelayAuthoRuleParameterSet,
+            HelpMessage = "WcfRelay Name.")]
+        [ValidateNotNullOrEmpty]
+        public string WcfRelay { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2, ParameterSetName = HybridConnectionAuthoRuleParameterSet,
+            HelpMessage = "HybridConnection Name.")]
+        [ValidateNotNullOrEmpty]
+        public string HybridConnection { get; set; }
+
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "AuthorizationRule Name.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = true,
+            Position = 4, 
+            ParameterSetName = AuthoRuleInputObjectParameterSet,
+            HelpMessage = "Relay AuthorizationRule Object.")]
+        [Parameter(ParameterSetName = NamespaceAuthoRuleParameterSet)]
+        [Parameter(ParameterSetName = WcfRelayAuthoRuleParameterSet)]
+        [Parameter(ParameterSetName = HybridConnectionAuthoRuleParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public AuthorizationRuleAttributes InputObject { get; set; }
+
+        [Parameter(Mandatory = true,
+            Position = 4,
+            ParameterSetName = AuthoRulePropertiesParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Rights, e.g.  @(\"Listen\",\"Send\",\"Manage\")")]
+        [Parameter(ParameterSetName = NamespaceAuthoRuleParameterSet)]
+        [Parameter(ParameterSetName = WcfRelayAuthoRuleParameterSet)]
+        [Parameter(ParameterSetName = HybridConnectionAuthoRuleParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string[] Rights { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            AuthorizationRuleAttributes sasRule = new AuthorizationRuleAttributes();
+
+            if (InputObject != null)
+            {
+                sasRule = InputObject;
+            }
+            else
+            {
+                sasRule.Rights = new List<string>();
+                if (Rights != null && Rights.Length > 0)
+                    foreach (string test in Rights)
+                    {
+                        sasRule.Rights.Add(test);
+                    }
+            }
+
+            // update Namespace Authorization Rule
+            if (Namespace != null && WcfRelay == null && HybridConnection == null)
```

Comment:
Same comments as above - assuem that these comments apply to all further cmdlets